docs/book/src/tasks/experimental-features/runtime-sdk/implement-topology-mutation-hook.md

```diff
@@ -88,6 +88,7 @@ templates, the global variables and the template-specific variables. The respons
 ```yaml
 apiVersion: hooks.runtime.cluster.x-k8s.io/v1alpha1
```

Comment:
Not sure where it fits, but should we also mention somewhere that settings can be set as part of the external patch config in the ClusterClass? (+ that those override the ones from ExtensionConfig)

Maybe in the ClusterClass YAML in `20220330-topology-mutation-hook.md`?